src/chrome/chromeConnection.ts

```diff
@@ -29,7 +30,7 @@ export interface ITarget {
 }
 
 export type ITargetFilter = (target: ITarget) => boolean;
-export interface ITargetDiscoveryStrategy {
+export interface ITargetDiscoveryStrategy extends ObservableEvents {
```

Comment:
Should it extend ObservableEvents ? Or let the concrete class implement both?

Reply:
Done